target_chains/ethereum/contracts/contracts/entropy/EntropyUpgradable.sol

```diff
@@ -103,4 +103,8 @@ contract EntropyUpgradable is
     function entropyUpgradableMagic() public pure returns (uint32) {
         return 0x66697265;
     }
+
+    function version() public pure returns (string memory) {
```

Comment:
nit: use `0.1.0` for the initial version. [ref](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase)